src/main/java/mate/academy/internetshop/dao/BucketDao.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.internetshop.dao;
+
+import mate.academy.internetshop.model.Bucket;
+
+import java.util.Optional;
+
+public interface BucketDao {
+    Bucket create(Bucket bucket);
+
+    Optional<Bucket> get(Long id);
+
+    Bucket update(Bucket bucket);
+
+    boolean delete(Bucket bucket);
+
+    boolean delete(Long bucketId);
```

Comment:
```suggestion
    boolean deleteById(Long bucketId);
```
краще заміни так у всіх класах

Reply:
done